src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -191,6 +191,12 @@ else()
     #     )
     endif ()
 
+    # Additional requirements for System.Security.Cryptography.Native.OpenSsl
+    message("FEATURE_DISTRO_AGNOSTIC_SSL: ${FEATURE_DISTRO_AGNOSTIC_SSL}")
```

Comment:
A nit - can you please remove this message?

Reply:
Done!